pool/connection_pool_test.go

```diff
@@ -3495,6 +3503,39 @@ func runTestMain(m *testing.M) int {
 	return m.Run()
 }
 
+func TestConnectionPool_GetInfo(t *testing.T) {
+	t.Run("equal instance info", func(t *testing.T) {
```

Comment:
Let's not make these additional levels of nesting here.
```suggestion
func TestConnectionPool_GetInfo_equal_instance_info(t *testing.T) {
```

Reply:
done